src/python_testing/matter_testing_infrastructure/matter/testing/matter_testing.py

```
@@ -2429,6 +2430,35 @@ async def commission_ntl_device(self, setup_payload: SetupPayload) -> bool:
         self._dut_confirmed_available = self._dut_confirmed_available or result
         return result
 
+    async def find_or_establish_pase_session_over_ntl(self, setup_payload: SetupPayload, node_id: int) -> DeviceProxyWrapper | None:
```

Flake8 (7.3.0)

[error] undefined name 'DeviceProxyWrapper'

(F821)